chrome/content/zotero/xpcom/data/collection.js

```diff
@@ -23,7 +23,7 @@
     ***** END LICENSE BLOCK *****
 */
 
-Zotero.Collection = function() {
+Zotero.Collection = function(name) {
```

Comment:
I've been trying to make constructors take required parameters, since they always need to be set anyway. libraryID defaults to user library, so it's not technically required.